kubernetes/running-single-container.md

```diff
@@ -0,0 +1,118 @@
+# Running a single container
+
+[comment]: # (TODO: Environment Setup Guide / Reference?)
+
+Let's say you just want to start one *one* container with Kubernetes. How would you do that?
+
+## Start a deployment
+
+First you need to define a deployment. This can be done a few ways, the fastest is with `kubectl run $DEPLOYMENT_NAME --image=$IMAGE --port=$PORT --replicas=$NUM_REPLICAS`. For example:
+
+```
+$ kubectl run webserver --image=nginx --port=80 --replicas=1
+deployment "webserver" created
+$ kubect get deployments
+NAME        DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+webserver   1         1         1            1           33s
+$ kubect get pods
```

Comment:
Might be a good idea to split out the pods from this code block, so we can add a sentence in front of it.

> ```
$ kubectl run webserver --image=nginx --port=80 --replicas=1
deployment "webserver" created
$ kubect get deployments
NAME        DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
webserver   1         1         1            1           33s
```

> As you can see, our deployment started a single copy of the container. To view each container instance, called a pod, run:

> ```
$ kubect get pods
NAME                         READY     STATUS    RESTARTS   AGE
webserver-3274421635-a4al1   1/1       Running   0          35m